api/room_resource/schema.py

```diff
@@ -26,6 +30,26 @@ def mutate(self, info, **kwargs):
         return CreateResource(resource=resource)
 
 
+class UpdateRoomResource(graphene.Mutation):
+    class Arguments:
+        name = graphene.String()
+        room_id = graphene.Int()
+        resource_id = graphene.Int()
+    resource = graphene.Field(Resource)
+
+    def mutate(self, info, resource_id, **kwargs):
+        validate_empty_fields(**kwargs)
+        query = Resource.get_query(info)
+        exact_resource = query.filter(ResourceModel.id == resource_id).first()
+
+        if not exact_resource:
+            raise GraphQLError("ResourceId not Found")
```

Comment:
What do you think about changing *ResourceId not Found* to *Resource not Found*? 🤔

Reply:
Thanks for the feedback. I think they are both appropriate. I just wanted to notify the `front-end dev` what exactly is wrong with their mutation.

Reply:
Ok, I get now!